### ISSUE TITLE
BCTools runs on wrong web pages.

### ISSUE THREAD

**Issue by misentle** [Dutchie322/bclub-tools#34]
BCTools runs on each page of www.bondageprojects.com, resulting in many console errors when playing college/poker. 
Replacing matching links for content script and permissions with these should fix it:
``` 
    "https://www.bondageprojects.com/college/*/BondageClub/",
    "https://bondageprojects.com/college/*/BondageClub/",
    "https://www.bondageprojects.elementfx.com/*/BondageClub/",
    "https://bondageprojects.elementfx.com/*/BondageClub/",
    "http://www.bondageprojects.com/college/*/BondageClub/",
    "http://bondageprojects.com/college/*/BondageClub/",
    "http://www.bondageprojects.elementfx.com/*/BondageClub/",
    "http://bondageprojects.elementfx.com/*/BondageClub/"
```

**Comment by Dutchie322** [comment]
The extension should check if it can find the game on the current page, and if not, should disable itself automatically. This was done to support beta and cheater versions, as "BondageClub" in the URL is replaced with a random number.

I see that it throws a lot of errors and keeps throwing "ChatRoomSpace is not defined" even after it couldn't find the ServerSocket variable. I'll look into making it a bit more elegant.

**Comment by Dutchie322** [comment]
A new version has been published for review on the Chrome web store: v0.5.0. I've rewritten the way the game is detected and the extension should not longer attempt to run on any page other than the club itself.
